tests/unit/backend/corpora/common/test_authorizer.py

```diff
@@ -1,23 +1,24 @@
+import json
 import os
 import unittest
 
 import requests
 from chalice import UnauthorizedError
 
 from backend.corpora.common.authorizer import assert_authorized
+from backend.corpora.common.utils.aws_secret import AwsSecret
 
-if not os.getenv("DEPLOYMENT_STAGE"):  # noqa
-    os.environ["DEPLOYMENT_STAGE"] = "test"  # noqa
 
-
-@unittest.skipIf(os.getenv("DEPLOYMENT_STAGE") != "test", "DEPLOYMENT_STAGE not 'test'")
+@unittest.skipIf(os.getenv("DEPLOYMENT_STAGE"), "DEPLOYMENT_STAGE not set")
 class TestAuthorizer(unittest.TestCase):
+
     @classmethod
     def setUpClass(cls):
-        cls.auth0_secret = {}
+        secret_name = "corpora/cicd/dev/auth0-secret"
+        # using the same secret for all non production stages.
```

Comment:
For readability, please move this comment to the line before the `secret_name` definition (i.e. swap lines 17 and 18). As it stands today, it seems as though the comment refers to the lines below.